Model/User.php

```diff
@@ -180,10 +180,7 @@ public function addRole($role)
         return $this;
     }
 
-    /**
-     * @return void
-     */
-    public function eraseCredentials()
+    public function eraseCredentials(): void
```

Comment:
this is a BC break for child classes. If this is required to support Symfony 7, it means we will have to release it as a new major version (in which case we might want to first do a new minor version to ensure that we have the appropriate `@return` everywhere so that our new major version can switch to native return types everywhere)

Reply:
I can see you added the return types in https://github.com/FriendsOfSymfony/FOSUserBundle/pull/3069 but never released it. Can you do that? 

Should I go ahead and add the native return types in this PR and then we could get new major version out or should that be separate PR?